lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -84,8 +85,19 @@ class CourseEnrollmentSerializer(serializers.ModelSerializer):
     course = CourseOverviewField(source="course_overview", read_only=True)
     certificate = serializers.SerializerMethodField()
 
+    @classmethod
+    def get_course_catalog_run(cls, user, course_keys):
```

Comment:
Nit: suggest renaming this to `get_catalog_course_runs` so its name is consistent with the `catalog_course_run` name already chosen elsewhere.